pkgs/development/interpreters/python/cpython/default.nix

```diff
@@ -389,7 +389,10 @@ in with passthru; stdenv.mkDerivation {
     done
     touch $out/lib/${libPrefix}/test/__init__.py
 
-    ln -s "$out/include/${executable}m" "$out/include/${executable}"
+    # This is gone in Python ≥ 3.8
+    if [ -e "$out/include/${executable}m" ]; then
```

Comment:
maybe we can fix `passthru.executable` instead

Reply:
I am not sure what you mean. passthru.executable seems to be correct to me.